apps/documentation/docs/fresco/deployment/advanced.en.mdx

```diff
@@ -1,7 +1,267 @@
 ---
 title: Advanced Deployment
-date: 13th March 2024
+date: 9th July 2024
 wip: true
 ---
 
-Coming soon!
+<SummaryCard duration="60 minutes">
+
+<SummarySection>
+
+For most users, the default [deployment process](./guide) will be sufficient. However, if your circumstances require using different cloud infrastructure or a VPS, you can do so by using [Docker](https://www.docker.com/).
+
+</SummarySection>
+
+<PrerequisitesSection>
+
+This guide is intended for advanced, technical users who are already familiar with Docker and cloud infrastructure. It provides a high-level overview of the steps required to deploy Fresco using Docker, but does not provide detailed instructions for each step.
+
+If you are not already familiar with these technologies, we recommend following our [default deployment guide](./guide).
+
+</PrerequisitesSection>
+</SummaryCard>
+
+<Tipbox danger>
+  Because of the nature of self-hosting Dockerized applications on cloud
+  infrastructure, you are responsible for the security and maintenance of your
+  own deployment! The Network Canvas project team by design has no access to
+  your data or infrastructure.
+</Tipbox>
+
+## Configure Hosting Infrastructure
+
+### Finding Hosting Services
+
+For advanced deployment, you will need to find a cloud service or VPS provider that supports Docker. Here are a few options:
+
+- [Digial Ocean](https://www.digitalocean.com/)
+- [AWS](https://aws.amazon.com/)
+- Your own VPS
+
+### Set up Docker Environment
+
+Ensure that you have Docker installed on your server. Each hosting provider will have different steps for setting up Docker, so please refer to their documentation for more information.
+
+- [Digital Ocean Docker Installation](https://www.digitalocean.com/community/tutorial-collections/how-to-install-and-use-docker)
+- [AWS Docker Installation](https://aws.amazon.com/docker/)
+
+## Prepare Fresco Repository
+
+Clone the [Fresco repository](https://github.com/complexdatacollective/Fresco) to your server. You can do this by running the following command:
+
+```bash
+git clone https://github.com/complexdatacollective/Fresco.git
+```
+
```

Comment:
We don't need this either, since the actual workflow will just be to pull straight from the container registry.

Replace this with a section on how to pull from the container registry, and a description of our tags (we will have `latest` and `v1.2.3` style tags).